CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 ## [Unreleased]
 
+### Changed
+
+- Bump grape to v1.5.0 - It speeds up the processing of POST requests for nested problems (up to 20 times faster)
+- Bump grape-swagger to v1.3.0 - The documentation is now correctly generated. It allows SDK generation.
+- The fields `costs` introduced in v0.1.5 is renamed `cost_details` to avoid confusion with field `cost`
+- Bump VROOM to v1.8.0 and integrates the features integrated since v1.3.0
+- Bump OR-Tools v7.8
+- VROOM were previously always called synchronously, it is now reserved to a set of effective `router_mode` (:car, :truck_medium) within in a limit of points (<200).
+
+### Fixed
+
+- `unassigneds` output were in some cases returning the key `shipment_id` instead of `pickup_shipment_id` and `delivery_shipment_id`
```

Comment:
```suggestion
- `unassigned` output were in some cases returning the key `shipment_id` instead of `pickup_shipment_id` and `delivery_shipment_id`
```